.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I keep this file as a reference on how to attach to the hardhat process and debug using vscode.